frontstage/views/secure_messaging.py

```diff
@@ -96,10 +97,10 @@ def messages_get(session, label="INBOX"):
     """Gets users messages"""
     party_id = session['party_id']
     loggerb = logger.bind(party_id=party_id, label=label)
-    messages = get_messages(label, loggerb)
-    unread_msg_total = get_unread_message_total(loggerb)
+    messages = get_messages_list(label, loggerb)
+    unread_msg_total = messages.get('total')
```

Comment:
Possibly rename 'total' to 'total_unread' in the frontage_api as the context has now changed from only unread labels to message list. 
Or rename the response in SM but that would be a breaking change if other apis are using the labels endpoint.
Or we can add that attribute in the message list in SM message list response instead of having to call another endpoint.

Reply:
I agree with that change, we can make it in the next update we make to the frontstage-api